src/parsers.php

```diff
@@ -475,10 +477,8 @@ function isFallible() : bool
     };
 }
 
-const
-    CONSUME_DO_TRIM = 0x10,
-    CONSUME_NO_TRIM = 0x01
-;
+if(!defined(__NAMESPACE__."\\CONSUME_DO_TRIM")) define(__NAMESPACE__."\\CONSUME_DO_TRIM", 0x10);
+if(!defined(__NAMESPACE__."\\CONSUME_NO_TRIM")) define(__NAMESPACE__."\\CONSUME_NO_TRIM", 0x01);
```

Comment:
Ok, so we won't need the conditional definitions of constants anymore... but we may still want to have the IDE stubs on a separate PR.

I would also fire a bug on your IDE support channel. IDEs should be able to recognize constants as they are part of PHP.